src/Form/GCSettingsForm.php

```diff
@@ -144,6 +144,20 @@ public function buildForm(array $form, FormStateInterface $form_state): array {
       '#placeholder' => '/path/to/page',
     ];
 
+    $form['app_settings']['live_stream_empty_text'] = [
+      '#type' => 'textfield',
+      '#title' => 'Display text when no livestream is scheduled',
+      '#default_value' => $config->get('live_stream_empty_text'),
+      '#required' => TRUE,
+    ];
+
+    $form['app_settings']['virtual_meeting_empty_text'] = [
+      '#type' => 'textfield',
+      '#title' => 'Display text when no event is scheduled',
+      '#default_value' => $config->get('virtual_meeting_empty_text'),
+      '#required' => TRUE,
```

Comment:
if it's required then checking empty text above makes no sense, right?
So we need to have an upgrade path in place to provide a default value and avoid having ternary operators logic